### ISSUE TITLE
Interpolation Issues v1.061 Roman #3 - advance width

### ISSUE THREAD

**Issue by kenmcd** [psb1558/Junicode-font#190]
Interpolation Issues v1.061 Roman 3 - advance width

File: JunicodeTwoBetaVF-Roman.ttf, v1.061 beta, 2023-07-10

Glyphs: /yloop/v.alt2/c.highoverline/d.highoverline/i.highoverline/j.highoverline/l.highoverline/m.highoverline/v.highoverline/x.highoverline/thorn.medhighoverline.loclENG/uF0004/uF0008/uF000C/uF000E/uF0010/uF0018/uF001A

Some of the advance widths, and/or side bearings, appear to be excessive when increasing the Enlarge axis.

![2023-07-12_19-19-39](https://github.com/psb1558/Junicode-font/assets/844920/af55088d-7016-4355-9daf-20886d09308d)

EDIT: hmmm... just realized - they are all lowercase - shouldn't these be enlarging?

p.s. gotta go, will have a few more tomorrow...

**Comment by psb1558** [comment]
Thanks for this. The Enlarge axis is intended for use with letters that can come in sentence- or word-initial position—broadly, the relatively few "enlarged" characters identified by MUFI (though I have added a few more). So these shouldn't enlarge, but when they don't, their Enlarged masters should be identical to their Regular masters. I have fixed these, and I'll write a script to look for others.

What program is your screenshot from?
